README.md

```diff
@@ -4,7 +4,7 @@
 * Alex B - [ProjectName]()
 * Braden S - [ProjectName]()
 * Eli P - [ProjectName]()
-* Isiah W - [ProjectName]() 
+* Isiah W - [Animal Pound]() 
```

Comment:
This still needs to link to your markdown file, which goes between the `(  )`. You should also change the name of the template.